jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java

```diff
@@ -131,7 +131,7 @@ public JdrReport collect() throws OperationFailedException {
                 ROOT_LOGGER.debugf(message);
                 skips.append(message);
                 PrintWriter pw = new PrintWriter(new StringWriter());
-                t.printStackTrace(pw);
```

Comment:
https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html#printStackTrace-java.io.PrintWriter-

Stack trace is printed to string writer and added to StringBuilder skips